integration_tests/smoke/smoke.spec.ts

```diff
@@ -18,13 +20,14 @@ context('Smoke test', () => {
       .submitForm<ReviewRecipientsPage>(ReviewRecipientsPage)
       .hasRecipientNamesExactly('lsj-smoketest')
       .hasPrisonNamesExactly('HMP Stocken')
-    const printCoversheetsPage = reviewRecipientsPage //
+    reviewRecipientsPage //
       .prepareBarcodes()
       .continueToCoversheet()
       .submitHavingSelectedDlEnvelopeSize()
-    printCoversheetsPage.smokeTestBarcode().then(smokeTestBarcode => {
-      cy.task('setSmokeTestBarcode', smokeTestBarcode)
-    })
+      .smokeTestBarcode()
+      .then(smokeTestBarcode => {
+        cy.task('setSmokeTestBarcode', smokeTestBarcode)
+      })
```

Comment:
The additional step to save the barcode within Cypress seemed to be triggering an additional page load test in Cypress which was causing the flakiness. This additional page load is unnecessary because the step `submitHavingSleectedDlEnvelopeSize` already checks we are on the Print Coversheet page.

Reply:
Wow, that was a subtle bug, but well done for tracking it down 👍